ios/FluentUI.Demo/FluentUI.Demo/Demos/TabBarViewDemoController.swift

```diff
@@ -61,7 +61,9 @@ class TabBarViewDemoController: DemoController {
 
     private func setupTabBarView() {
         // remove the old tab bar View
+        var openIsUnread = true
```

Comment:
boolean names should always start with clear verb  https://microsoft.github.io/swift-guide/Naming.html#variables-and-properties